crates/rpc/src/handlers/config.rs

```diff
@@ -17,12 +17,46 @@ pub struct DepositContract {
     chain_id: u64,
     address: Address,
 }
+#[derive(Serialize, Deserialize, Default)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub struct SpecConfig {
+    deposit_contract_address: String,
+    #[serde(with = "serde_utils::quoted_u64")]
+    deposit_network_id: u64,
+    domain_aggregate_and_proof: String,
+    #[serde(with = "serde_utils::quoted_u64")]
+    inactivity_penalty_quotient: u64,
+}
 
 impl DepositContract {
     pub fn new(chain_id: u64, address: Address) -> Self {
         Self { chain_id, address }
     }
 }
+impl SpecConfig {
+    pub fn new(
+        deposit_contract_address: String,
+        deposit_network_id: u64,
+        domain_aggregate_and_proof: String,
+        inactivity_penalty_quotient: u64,
+    ) -> Self {
+        Self {
+            deposit_contract_address,
+            deposit_network_id,
+            domain_aggregate_and_proof,
+            inactivity_penalty_quotient,
+        }
+    }
+
+    pub fn from_network_spec(network_spec: &NetworkSpec) -> Self {
+        Self::new(
+            network_spec.deposit_contract_address.to_string(),
+            network_spec.network.chain_id(),
+            network_spec.domain_aggregate_and_proof.clone(),
+            network_spec.inactivity_penalty_quotient,
+        )
+    }
+}
```

Comment:
```suggestion
impl From<Arc<NetworkSpec>> for  SpecConfig {
    etc etc {
           Self {
            deposit_contract_address: network_spec.deposit_contract_address,
            deposit_network_id: network_spec.network.chain_id(),
            domain_aggregate_and_proof: DOMAIN_AGGREGATE_AND_PROOF,
            inactivity_penalty_quotient: INACTIVITY_PENALTY_QUOTIENT_BELLATRIX,
      }
    }
}
```

You should use the from trait and delete this impl